packages/odyssey-react-mui/src/labs/DatePickers/dateTimePickerTheme.ts

```diff
@@ -75,16 +83,38 @@ const theme: ThemeOptions = {
     },
     MuiPickersLayout: {
       styleOverrides: {
+        root: () => ({
+          width: DateTimePickerSize.maxWidth,
+          maxWidth: DateTimePickerSize.maxWidth,
+        }),
         contentWrapper: ({ theme }) => ({
           padding: theme.spacing(3),
           gridTemplateColumns: "1fr 16px auto",
         }),
       },
     },
+    MuiMultiSectionDigitalClock: {
+      styleOverrides: {
+        root: {
+          maxHeight: `${DateTimePickerComponentSize.dateCalendarMaxHeight} !important`,
+          // Target all possible variations at once with the highest specificity
+          "&.MuiMultiSectionDigitalClock-root": {
+            maxHeight: `${DateTimePickerComponentSize.dateCalendarMaxHeight} !important`,
+          },
+          "& .MuiMultiSectionDigitalClockSection-root": {
+            maxHeight: `${DateTimePickerComponentSize.dateCalendarMaxHeight} !important`,
+          },
+          "&.MuiDigitalClock-root": {
+            maxHeight: `${DateTimePickerComponentSize.dateCalendarMaxHeight} !important`,
+          },
+        },
```

Comment:
We shouldn't use `!important` in Odyssey especially when we control the styles. Is there a reason we can't just set these normally?

We shouldn't be using `MuiDigitalClock-root` directly when we can import the class name directly from MUI.

There are some examples of that in Odyssey in `theme/components.tsx`. Older examples used direct class names, but we shouldn't do that.